src/components/color_picker/color_palette_display/_color_palette_display.scss

```
@@ -0,0 +1,24 @@
+.euiColorPaletteDisplay {
+  display: flex;
+  flex-direction: row;
+  overflow: hidden;
+  height: $euiSizeS;
+}
+
+$sizes: (
```

This needs to be name-spaced because this will be included in the whole theme file. It should also live in it's own `variables` file.
```suggestion
$euiColorPaletteDisplaySizes: (
```